form-validation.js

```diff
@@ -0,0 +1,18 @@
+function validation() {
+  const form = document.getElementById('form');
+  const email = form.elements.email.value;
+  const message = document.getElementById('message');
+  message.innerHTML = '';
+  const re = /@/g;
+  if (re.test(String(email).toLowerCase()) === true) {
+    if (email !== email.toLowerCase()) {
+      form.addEventListener('submit', (event) => {
+        event.preventDefault();
+      });
+      message.innerHTML = '* Make sure your email is in lower case.';
+    } else {
+      form.submit();
+    }
+  }
+}
+validation();
```

Comment:
- [ ] For now, you have mentioned to submit the form if the email is in the correct form and follow the requirement but there is an issue with this, if I just enter the email correctly without filling any other input field and trying to submit, it is submitting which is not a good behaviour of form. Please fix this. :slightly_smiling_face:

Reply:
Requested change implemented 🤓